src/plugins/index_pattern_field_editor/public/lib/documentation.ts

```diff
@@ -11,11 +11,10 @@ import { DocLinksStart } from 'src/core/public';
 export const getLinks = (docLinks: DocLinksStart) => {
   const { DOC_LINK_VERSION, ELASTIC_WEBSITE_URL } = docLinks;
   const docsBase = `${ELASTIC_WEBSITE_URL}guide/en`;
-  const esDocsBase = `${docsBase}/elasticsearch/reference/${DOC_LINK_VERSION}`;
   const painlessDocsBase = `${docsBase}/elasticsearch/painless/${DOC_LINK_VERSION}`;
 
   return {
-    runtimePainless: `${esDocsBase}/runtime.html#runtime-mapping-fields`,
+    runtimePainless: `${docsBase}/kibana/master/managing-index-patterns.html#runtime-fields`,
```

Comment:
I think instead of hard-coding a version in here, it'd be preferrable to define a const instead. For example:

>const kibanaDocsBase = ${docsBase}/kibana/${DOC_LINK_VERSION}`;

Then return the URL like this:
>    runtimePainless: `${kibanaDocsBase}/managing-index-patterns.html#runtime-fields`,